docs/services/motion/_index.md

```diff
@@ -98,7 +98,7 @@ By default, motion is unconstrained with the exception of obstacle avoidance.
 
 **Returns:**
 
-- [(bool)](https://docs.python.org/3/library/functions.html#bool): Whether the move was successful.
+- [(bool)](https://docs.python.org/3/library/stdtypes.html#bltin-boolean-values): Whether the move was successful.
```

Comment:
Sorry if this is a silly question but for my own understanding and future reference, why use this link versus the other one?

Reply:
NBD between either of them, just wanted to make them all match and figured the types docs was better than the bool function docs (I assume that's for casting?)!

Reply:
Sounds good, makes sense! I think I just started using one for all of them and kept doing it (without particular intent) but happy to switch going forward :D